pages/index.js

```diff
@@ -1,13 +1,19 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useState } from "react";
 
 import LoginForm from "../src/components/login-form";
+import VoteForm from "../src/components/vote-form";
+
+import styled from "styled-components";
+import Axios from "axios";
 
 export default function Home() {
+  const [login, setLogin] = useState(false);
```

Comment:
```suggestion
  const [isLoggedIn, setLoggedIn] = useState(false);
```
boolean값을 표현하는 변수라 이름에 is를 붙여주면 더 좋을 것 같아요!